src/ontology/mondo-edit.obo

```diff
@@ -615854,6 +615854,16 @@ relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/
 property_value: http://purl.org/dc/terms/creator https://orcid.org/0000-0002-7638-4659
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/8866" xsd:anyURI
 
+[Term]
+id: MONDO:1060110
+name: HDAC4-related haploinsufficiency syndrome
+def: "A disorder caused by haploinsufficiency of HDAC4 and is characterized by brachydactyly type E, variable mild to moderate intellectual disability, seizures, autism spectrum disorder, short stature, obesity, and facial dysmorphism. These features may also be observed in the context of the larger 2q37 deletion syndrome, but this term is used to describe individuals with variants in HDAC4 alone. Individuals with the gain of functions variants typically do not have brachydactyly and do have more consistent intellectual disability." [https://www.clinicalgenome.org/affiliation/40006/, PMID:33537682]
+synonym: "brachydactyly mental retardation syndrome" BROAD [https://www.clinicalgenome.org/affiliation/40006/, OMIM:600430]
```

Comment:
for MR, we add the synonym type 'A synonym that is historic and discouraged':

![image](https://github.com/user-attachments/assets/da8ae5cd-c74b-4fcd-98d9-4262717c86b0)

it will look like this:
![image](https://github.com/user-attachments/assets/0e81e480-9edd-4edc-80ce-090b583cc5ac)

Then add the broad synonym 'brachydactyly intellectual disability syndrome' instead. We replace MR with intellectual disability.

Reply:
@MeeSiing can you please add the broad synonym 'brachydactyly intellectual disability syndrome' instead. We replace MR with intellectual disability.

Reply:
thanks!